core/src/avm2/globals/flash/display/MovieClip.as

```diff
@@ -0,0 +1,27 @@
+package flash.display {
+    public dynamic class MovieClip extends Sprite {
+        public function MovieClip() {}
```

Comment:
Is movieclip allowed to be constructed normally?

Reply:
Yes: https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/display/MovieClip.html#MovieClip()